app/views/profile/users/show.html.erb

```diff
@@ -1,7 +1,8 @@
-<h1>User Profile</h1>
-  <%= link_to "Edit Profile", profile_edit_path, :class => "btn btn-outline-success" %>
-  <%= link_to "Favorites", root_path, :class => "btn btn-outline-success" %>
-  <%= link_to "My Friends", profile_friends_path, :class => "btn btn-outline-success" %>
+<h2>User Profile</h2>
+
+<%= link_to "Edit Profile", profile_edit_path, :class => "btn btn-outline-success" %> |
+<%= link_to "Favorites", profile_favorites_path, :class => "btn btn-outline-success" %> |
```

Comment:
Nice, this is one of the errors I found earlier when looking through the app on Heroku